core/src/js/components/collection/card-history.component.ts

```diff
@@ -13,6 +13,7 @@ import { AbstractSubscriptionComponent } from '../abstract-subscription.componen
 		`../../../css/global/forms.scss`,
 		`../../../css/global/toggle.scss`,
 		`../../../css/component/collection/card-history.component.scss`,
+		`../../../css/component/duels/desktop/filters/_duels-filters.component.scss`,
```

Comment:
In that case, maybe we should move this kind of little toogles with right aligned label to a separate component, because it is used in many places ("Use class icon", "Show archived", "Hide low data", "Show only main packs", "Show names" and so on)

Reply:
That could be an option indeed. Maybe either under `global`, or at the `components` root, or even in a new folder.

Reply:
What would you like me to name this component?